src/components/Footer.js

```diff
@@ -139,171 +140,249 @@ const ComBox = styled.div`
     padding: 16px 16px 16px 16px;
 `;
 
+const ComBoxTitle = styled.h3`
```

Comment:
Coding standards make sure every styled element is Styled... e.g. StyledComBoxTitle not ComBoxTitle. More below but will just comment here